instrumentation/rack/test/opentelemetry/instrumentation/rack/middlewares/tracer_middleware_test.rb

```diff
@@ -311,6 +311,17 @@
     end
   end
 
+  describe 'config[:response_propagators]' do
+    describe 'with ResponseTextMapPropagator' do
+      let(:config) { default_config.merge(response_propagators: [OpenTelemetry::Trace::Propagation::TraceContext::ResponseTextMapPropagator.new]) }
+
+      it 'injects the traceresponse header' do
+        res = Rack::MockRequest.new(rack_builder).get('/ping', env)
+        _(res.headers).must_include('traceresponse')
+      end
+    end
+  end
+
```

Comment:
Can we add tests for:

- when this option is not configured? 
- when the propagator throws an exception?